### PR TITLE
Pypi

### PR DESCRIPTION
**Pull request by kattni**
Following from https://github.com/adafruit/circuitpython/issues/979

### CODE REVIEW

**Review by brennen — CHANGES_REQUESTED** [review]
Looks like this is missing the `autodoc_mock_imports` removal; otherwise good I think.

**Comment by kattni** [comment]
Ah I did fix it and push it, but I didn't notice that it rejected the push because I'm not even with this branch. I'll get it fixed.

File: .travis.yml [hunk 1]
```diff
@@ -1,35 +1,34 @@
...
 python:
-    - "3.6"
-
+- '3.6'
 cache:
-    pip: true
-
+  pip: true
 deploy:
   provider: releases
```
Comment (tannewt) [inline .travis.yml:9]: There should be a - here before provider. I think thats why the release files are missing.

Comment (kattni) [reply .travis.yml:9]: There's no `- here` in the instructions or in any of the `.travis.yml` files I can find, including libs I didn't PR.

Please confirm this is necessary. 

If so, we need to update the instructions and I will go back through all the libs we've done and add it.

Comment (kattni) [reply .travis.yml:9]: Oh you mean a `-` before provider. I see. It apparently needs to be added each time, it's not automatic. I'll go back through everything and make sure it's there in the ones done before I started as well.

It's also not clear in the instructions that it needs to be added. It says "add the PyPi section including password". I didn't think to verify the rest, the instructions make it sound like the reformat did everything needed to the rest of it.

**Pull request merged**